stdlib/moreLabels.mli

```diff
@@ -1321,12 +1321,97 @@ module Set : sig
       val of_seq : elt Seq.t -> t
       (** Build a set from the given bindings
           @since 4.07 *)
+
+      (** Enumerations of the elements of a set.
+
+          An enumeration is an immutable data structure: its implementation
+          involves no side effects. It represents the sequence of the elements
```

Comment:
I find the immutability of enumeration obvious given the API and the fact that sets are already persistent. The focus on being not-mutable is okay but slightly weird. I would just have written "An enumeration represents the persistent sequence of ...".